src/terminal.ts

```diff
@@ -196,6 +196,59 @@ export class FlatpakManifest {
     throw new Error('Failed to build application')
   }
 
+  /**
+  * Gets an array of commands for a autotools build
+  * - If the app is being rebuilt
+  *   - Configure with `configure`
+  * - Build with `make`
+  * - Install with `make install`
+  * @param  {string}     rebuild     Whether this is a rebuild
+  * @param  {string[]}   buildArgs   The build arguments
+  * @param  {string}     configOpts  The configuration options
+  */
+  getAutotoolsCommands(
+    rebuild: boolean,
+    buildArgs: string[],
+    configOpts: string
+  ): Command[] {
+    const commands: Command[] = []
+    if (!rebuild) {
+      commands.push(
+        new Command(
+          'flatpak',
+          [
+            'build',
+            ...buildArgs,
+            this.repoDir,
+            './configure',
+            '--prefix=/app',
+            configOpts,
+          ],
+          path.join(this.workspace),
+          this.isSandboxed
+        )
+      )
+    }
+    commands.push(
+      new Command(
+        'flatpak',
+        ['build', ...buildArgs, this.repoDir, 'make', '-p', '-n', '-s'],
+        path.join(this.workspace),
+        this.isSandboxed
+      )
+    )
+
+    commands.push(
+      new Command(
+        'flatpak',
+        ['build', ...buildArgs, this.repoDir, 'make', 'V=0', '-j5', 'install'],
```

Comment:
Do we we want to hardcode the number of builders to 5?

Reply:
Good point.
Honestly not sure why GNOME builder chooses five on my machine (pretty confident that no machine has an uneven number of cores).

Can we get the number of cores without an additional dependency?

Reply:
I just wonder if we care enough about one aspect of autotools that we want to add logic for that.

Reply:
>  Can we get the number of cores without an additional dependency?

Nodejs has definitely an API for this

>  I just wonder if we care enough about one aspect of autotools that we want to add logic for that.

Well, we will definitely have to do more integration if we want things to work as expected ^^

Reply:
Added https://nodejs.org/api/os.html#os_os_cpus